src/modules/brave.ts

```diff
@@ -87,6 +95,23 @@ export async function getRewardsPayoutRecordHistory(): Promise<
   );
 }
 
+export async function getMauDau(): Promise<MauDauResponse> {
+  if (!process.env.BRAVE_MAUDAU_URL) {
+    throw new Error("Environment variable BRAVE_MAUDAU_URL is not set");
+  }
+  debugLOG("Requesting MAU/DAU data from BRAVE_MAUDAU_URL");
+  const endpoint = process.env.BRAVE_MAUDAU_URL;
+  const response = await fetch(endpoint).then((res) => res.json()) as MauDauResponse;
+  // Round the figures to the nearest 100K
+  for (const date of Object.keys(response) as (keyof typeof response)[]) {
+    for (const key of Object.keys(response[date]) as (keyof typeof response[typeof date])[]) {

```

Comment:
Super minor, but any reason for not using `for...in` here?

Reply:
Old habit from earlier days; `for...in` would enumerate some properties unexpectedly when you were dealing with custom classes and inheritance. In this case, however, I think it should be fine to use `for...in`, and possibly even more concise. New commit incoming!